components/safety_board/include/SYS_Vehicle.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file SYS_Vehicle.h
+ * @brief  Header file for the Vehicle's Safety State Machine
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @version 0.1
+ * @date 2023-01-16
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "Types.h"
+
+
+/******************************************************************************
+ *                              D E F I N E S
+ ******************************************************************************/
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef enum
+{
+  IMD_STATUS = 0x00,
```

Comment:
enums should follow the same pattern and only diverge at the end. 

eg, this should be
```c
STATUS_INDEX_IMD = 0x00,
STATUS_INDEX_BMS,
```
etc.

Same comment on the other enums below